Sources/App/Commands/Alerting.swift

```diff
@@ -83,7 +83,6 @@ extension Alerting {
             self.updatedAt = build.updatedAt!
             self.builderVersion = build.builderVersion
             self.platform = build.platform
-            self.runnerId = build.runnerId
```

Comment:
We don't want to remove this from Alerting. The linux info can be helpful (and even the orka one could be if there's a stuck runner).

Reply:
OK, so we need to leave the field and just remove it user-facing?

Reply:
Yes, exactly.